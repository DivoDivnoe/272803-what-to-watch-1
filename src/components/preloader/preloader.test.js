import React from 'react';
import renderer from 'react-test-renderer';
import Preloader from './preloader.jsx';

describe(`Preloader component`, () => {
  it(`renders correctly`, () => {
    const tree = renderer.create(
        <Preloader />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
