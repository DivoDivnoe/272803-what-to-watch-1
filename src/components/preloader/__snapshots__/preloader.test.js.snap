// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Preloader component renders correctly 1`] = `
<div
  className="player__preloader"
  style={
    Object {
      "left": "50%",
      "position": "absolute",
      "top": "50%",
      "transform": "translate(-50%, -50%)",
    }
  }
>
  <img
    alt="preload"
    height="50"
    src="/img/preloader.gif"
    width="50"
  />
</div>
`;
